Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mambaorg/micromamba:1.4.9-bullseye-slim
+FROM mambaorg/micromamba:1.5.9-bookworm-slim
```

Comment:
Is micromamba still a thing? Perhaps this should now be https://quay.io/repository/condaforge/miniforge3

Reply:
I think so. I may be wrong here, but I believe that `mamba`/`micromamba` are still developed together on https://github.com/mamba-org/mamba.